.env-development

```diff
@@ -0,0 +1 @@
+PGDATABASE=development-data
```

Comment:
- These should be gitignored, no? Think you have some punctuation to edit, both in file names and for each database name

Reply:
- check your db files for what the db names should be, and how they're spelled

Reply:
I have  altered the .env file names to follow the convention laid out in the gitignore folder